kubectl/plan.sh

```diff
@@ -0,0 +1,60 @@
+pkg_name=kubectl
+pkg_origin=core
+pkg_license=('Apache 2.0')
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_version="1.6.3"
+pkg_source="https://storage.googleapis.com/kubernetes-release/release/v${pkg_version}/bin/linux/amd64/kubectl"
+pkg_shasum="2e9872fe2409deee257434a8c45976a4e1cc3f2e4e2a972fd85e86127a1f7cbd"
+pkg_build_deps=(core/curl)
+pkg_bin_dirs=(bin)
+
+do_begin() {
+  return 0
+}
+
+do_download() {
+    mkdir -p $HAB_CACHE_SRC_PATH/${pkg_dirname}
+
+    cd $HAB_CACHE_SRC_PATH/${pkg_dirname}
+    
+    curl -o ${pkg_dirname} -s ${pkg_source}
```

Comment:
You don't need to override this callback or `do_verify`, just `do_unpack`, then copy the download executable into the prefix.